pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -259,7 +259,7 @@ def _evaluate(self, model: LightningModule, dataloaders, max_batches: int, test_
                     continue
 
                 # stop short when on fast_dev_run (sets max_batch=1)
-                if batch_idx >= max_batches:
+                if batch_idx >= max_batches[dataloader_idx]:
```

Comment:
Here max_batches is a list of num_batches for each dataloader.

Reply:
that makes sense, that's what I had  in mind